src/ci/bin/test.inc.sh

```diff
@@ -709,7 +716,12 @@ cromwell::build::setup_centaur_environment() {
     cromwell::private::create_centaur_variables
     if [ "${CROMWELL_BUILD_CENTAUR_TYPE}" = "${CROMWELL_BUILD_CENTAUR_TYPE_ENGINE_UPGRADE}" ]; then
         cromwell::private::setup_prior_version_resources
-    fi
+    elif [ "${CROMWELL_BUILD_CENTAUR_TYPE}" = "${CROMWELL_BUILD_CENTAUR_TYPE_PAPI_UPGRADE}" ]; then
+        cromwell::private::setup_prior_version_resources
+        export CROMWELL_BUILD_CROMWELL_CONFIG="${CROMWELL_BUILD_RESOURCES_DIRECTORY}/papi_v1_37_papi_v2_upgrade.application.conf"
+    elif [ "${CROMWELL_BUILD_CENTAUR_TYPE}" = "${CROMWELL_BUILD_CENTAUR_TYPE_PAPI_UPGRADE_NEW_WORKFLOWS}" ]; then
+        export CROMWELL_BUILD_CROMWELL_CONFIG="${CROMWELL_BUILD_RESOURCES_DIRECTORY}/papi_v1_37_papi_v2_upgrade.application.conf"
```

Comment:
🤔 As `37` gets released, then `develop` moves to `38`, `39`, etc. -- will this still work as expected? Aka is this variable value "hard-coded" `_37_` still correct after the next release? There are other configs files with version numbers in their names. But there is bash code in `setup_prior_version_resources` than dynamically generates the current & previous version numbers, only detecting those config files for the correct version of the tests in `develop`.

Reply:
Discussed in person, leaving this as is for now and ticketing a decision on the future of this code. Perhaps all of this can just be deleted once v2 goes to production.